### ISSUE TITLE
Relative paths don't work

### ISSUE THREAD

**Issue by AliSoftware** [krzysztofzablocki/Insanity#4]
This doesn't work
```
$ Pods/Insanity/bin/insanity InsanityDemo Insanity/Templates/allEnums.stencil Insanity/Generated
'InsanityDemo' does not exist or is not readable.
```
But this works
```
$ Pods/Insanity/bin/insanity "$PWD"/InsanityDemo "$PWD"/Insanity/Templates/allEnums.stencil "$PWD"/Insanity/Generated
Scanning sources...
Found 3 types
Loading templates...
Loaded 1 templates
Generating code...
```

**Comment by krzysztofzablocki** [comment]
I'll correct it after work 👍

**Comment by krzysztofzablocki** [comment]
Fixed in `0.2.1`
